crawl-ref/source/webserver/process_handler.py

```diff
@@ -797,7 +802,7 @@ def _ttyrec_id_header(self): # type: () -> bytes
                  clrscr)
 
     def _on_process_end(self):
-        self.logger.info("Crawl terminated.")
+        self.logger.debug("Crawl PID %s terminated.", self.process.pid)
```

Comment:
not sure why this one is switched to `debug`?

Reply:
this prints every time a player ends their game -- it is log spam IMO